src/NServiceBus.AcceptanceTests/Exceptions/Handler_and_UowEnd_throws.cs

```diff
@@ -24,37 +24,11 @@ public void Should_receive_AggregateException_with_both_exceptions()
 
             Assert.AreEqual(typeof(HandlerException), context.InnerExceptionOneType);
             Assert.AreEqual(typeof(EndException), context.InnerExceptionTwoType);
-
-
-            StackTraceAssert.StartsWith(
```

Comment:
so the stack trace asserts are removed. I look forward to the eventual regression